models/evc.py

```diff
@@ -1480,6 +1623,7 @@ def handle_topology_update(self, switches: dict):
         # All intra-switch EVCs do not have current_path.
         # In case of inter-switch EVC and not current_path,
         # link_down should take care of deactivation.
+        return
```

Comment:
This `return` is here to avoid early activations so consistency checks can be tested.

Reply:
@Alopalao as we spoke in the meeting, if you could `git rebase` with this branch `fix/handle_switch_changes` of https://github.com/kytos-ng/mef_eline/pull/383, that'd be great. That way the linter error would also be gone. It was great how you managed to progress in the meantime.

Reply:
Rebased it and tested it, both are working well.

Reply:
Great. Certain `master` originated commits here that haven't been merged into #383 is making the review slightly more difficult, if you could talk to @Ktmi to merge `master` into his branch, that'd be great.

Reply:
@Alopalao, I was checking David's PR commits history he's already got the correct commits there:

```
commit c9d91ed51799ce56e76e937d92bcb9cae6e1a9fb
Author: Vinicius Arcanjo <vindasil@fiu.edu>
Date:   Tue Oct 31 17:50:48 2023 -0300

    feat: add set_vlan only if in_vlan_a != in_vlan_z when pushing UNI flow
    related to issue 389


```

and on `master` same sha1:

```
commit c9d91ed51799ce56e76e937d92bcb9cae6e1a9fb
Author: Vinicius Arcanjo <vindasil@fiu.edu>
Date:   Tue Oct 31 17:50:48 2023 -0300

    feat: add set_vlan only if in_vlan_a != in_vlan_z when pushing UNI flow
    related to issue 389
```

So you'll need to sort out why these commits got rewritten, if you need help with this let me know.